pkg/cloud/gcp/gcs_storage.go

```diff
@@ -43,6 +45,10 @@ const (
 	// CredentialsParam is the query parameter for the base64-encoded contents of
 	// the Google Application Credentials JSON file.
 	CredentialsParam = "CREDENTIALS"
+	// ServiceAccountParam is the email of the service account to impersonate.
+	// This service account should have the necessary access to the relevant cloud
+	// services.
+	ServiceAccountParam = "SERVICE_ACCOUNT"
```

Comment:
nit: I definitely think ServiceAccount is clearer than target but does this now give the impression that we are passing the service account that will be used to access cloud services? I guess in a way we are but should we be more explicit and call it `ASSUMED_SERVICE_ACCOUNT` or something? Might also be worth asking a few more folks for a bikeshed?

Reply:
Done, now we are using the param `ASSUME_ROLE` and not using an additional auth method